odybcl2fastq/Snakefile

```diff
@@ -128,11 +128,12 @@ rule count_10x_cmd:
         cmd="#!/bin/bash\n"
         cmd+="ulimit -u \$(ulimit -Hu)\n"
         cmd+="exit_code=0\n"
-        cmd+="mkdir -p {config[output_slurm]}{wildcards.run}/count\n"
+        cmd+="mkdir -p {config[output_slurm]}{wildcards.run}/count/{wildcards.sample}\n"
         cmd+="mkdir -p /scratch/{wildcards.run}_{wildcards.sample}_\$SLURM_JOB_ID\n"
         cmd+="cd /scratch/{wildcards.run}_{wildcards.sample}_\$SLURM_JOB_ID\n"
         cmd+="/usr/bin/time -v cellranger{config[atac]} count --id={wildcards.sample} $transcriptome --sample={wildcards.sample} --fastqs=$fastq_path --localmem=\$((9*\$(ulimit -m)/10000000)) --localcores=\$SLURM_JOB_CPUS_PER_NODE || exit_code=\$?\n\n"
-        cmd+="rsync --info=STATS -rtl --safe-links --perms --chmod=Dug=rwx,Fug=rw /scratch/{wildcards.run}_{wildcards.sample}_\$SLURM_JOB_ID/{wildcards.sample}/ {config[output_slurm]}{wildcards.run}/count/{wildcards.sample}/ || exit_code=\$((exit_code | \$?))\n"
+        cmd+="cp -p {wildcards.sample}/*.mri.tgz {config[output_slurm]}{wildcards.run}/count/{wildcards.sample}/ || exit_code=\$((exit_code | \$?))\n"
+        cmd+="[ \$exit_code -eq 0 ] && /usr/bin/time -v cp -Rp {wildcards.sample}/outs {config[output_slurm]}{wildcards.run}/count/{wildcards.sample}/ || exit_code=\$((exit_code | \$?))\n"
```

Comment:
Is the outs dir there even on failure?  If so it might be more straight forward just to always copy both tgz file and outs.

Reply:
The outs dir is not there on failure; I figured having partial output wouldn't be helpful, and any error messages (stderr, etc.) should be in the .mri.tgz file

Reply:
Or to clarify: the outs dir is in /scratch on failure, but wouldn't be useful to copy back to the `{config[output_slurm]}{wildcards.run}/count/{wildcards.sample}` directory